reactivesocket-tck-drivers/src/main/java/io/reactivesocket/tckdrivers/client/JavaClientDriver.java

```diff
@@ -0,0 +1,517 @@
+/*
+ * Copyright 2016 Facebook, Inc.
```

Comment:
I believe we agreed to put Netflix as copyright holder for submissions to this repo for simplicity.  cc @benjchristensen